src/eventcontroller/DefaultEventController.ts

```diff
@@ -15,39 +22,42 @@ import EventName from './EventName';
 import flattenEventAttributes from './flattenEventAttributes';
 import MeetingHistoryState from './MeetingHistoryState';
 
-export default class DefaultEventController implements EventController {
-  /** @internal */
+export default class DefaultEventController implements EventController, Destroyable {
   private static readonly UNAVAILABLE = 'Unavailable';
 
   // Use "ua-parser-js" over "detect-browser" to get more detailed information.
   // We can consider replacing "detect-browser" in DefaultBrowserBehavior.
-  /** @internal */
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   private parserResult: any;
-  /** @internal */
   private browserMajorVersion: string;
-  /** @internal */
   private meetingHistoryStates: { name: MeetingHistoryState; timestampMs: number }[] = [];
+  private observerSet: Set<EventObserver> = new Set<EventObserver>();
+
+  private logger: Logger;
+  private configuration: MeetingSessionConfiguration;
+  private _eventReporter: EventReporter;
 
   // Compute these once so we're not doing work on each event.
-  /** @internal */
```

Comment:
Sorry do not remember why this was added? Can you once check if Ok to remove this with Kyu?

Reply:
Checked with Kyu, previously it was needed so the doc won't have it, but checked local doc and seems to be fine without it. Will check live version once it's updated